wp-content/themes/engage/assets/scss/components/_research.scss

```diff
@@ -16,6 +16,32 @@
 	}
 }
 
+.aspect-ratio-box {
```

Comment:
Can you move `.aspect-ratio-box` related code into a more generic/utility type css file? It's not explicitly related to _research, so it makes more sense to put it somewhere that we'd find it if we were looking for it.